libcxx/include/__type_traits/operation_traits.h

```diff
@@ -18,8 +18,11 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-template <class _Pred, class _Lhs, class _Rhs>
-struct __is_trivial_plus_operation : false_type {};
+template <class _Operation, class _Canonical>
+struct __desugars_to : false_type {};
+
+template <class _Operation>
+struct __desugars_to<_Operation, _Operation> : true_type {};
```

Comment:
Now this one wouldn't actually make sense anymore, since `__desugars_to` would take an operation and a tag, not two operations.